comment_gh_pr.py

```diff
@@ -0,0 +1,8 @@
+from github import Github
+
+def comment_gh_pr(repo, pr, msg):
+      from os import environ
+      gh = Github(gh = Github(login_or_token=open(expanduser(repo_config.GH_TOKEN)).read().strip()))
```

Comment:
I do not like this, here it is making use of expanduser and repo_config without any ref to these. 
I would suggest keep comment-gh-pr as it is and just add extra function in github_utils.py e.g.

```
def comment_gh_pr(repo, pr, msg):
      gh = Github(login_or_token=get_gh_token(repo))
      repo = gh.get_repo(repo)
      pr   = repo.get_issue(pr)
      pr.create_comment(msg)  
```

Reply:
OK. I just wanted to share his idea of using your own comment_gh_pr so you can do additional things like set commit statuses.